ta/sks/include/sks_ta.h

```diff
@@ -0,0 +1,62 @@
+/* SPDX-License-Identifier: BSD-2-Clause */
+/*
+ * Copyright (c) 2018-2019, Linaro Limited
+ */
+
+#ifndef __SKS_TA_H__
+#define __SKS_TA_H__
+
+#include <sys/types.h>
+#include <stdint.h>
+#include <util.h>
+
+#define TA_SKS_UUID { 0xfd02c9da, 0x306c, 0x48c7, \
+			{ 0xa4, 0x9c, 0xbb, 0xd8, 0x27, 0xae, 0x86, 0xee } }
+
+/* SKS trusted application version information */
+#define SKS_VERSION_MAJOR		0
+#define SKS_VERSION_MINOR		0
```

Comment:
add 
```
#define SKS_VERSION_PATCH		0
```
for proper semantic versioning

Reply:
The versioning stuff is just a last resort. I hope that the intention of this is just a have one final life line.
Consider what happens if major version is increased, you could just as well change the uuid since the old clients cannot communicate with the TA any way.

Reply:
Changing the Master version does not mean that the API has completely changed. Plus, this in an internal API, not the external PKCS#11 API.